app/assets/stylesheets/application.scss

```diff
@@ -67,13 +67,8 @@ nav {
   color: darken($primary-color, 30%);
 }
 
-.checkbox {
-  margin: 20px;
-  padding: 6px;
-  color: #FF495F;
+#lp-background{
+  background: url("http://conference.cs.cityu.edu.hk/wise2010/images/Background.jpg") no-repeat;
```

Comment:
if this image goes down/gets renamed, you won't have a background image to show instead what you can do is add an image to

```css
#lp-background {
  background: image-url('background.jpg') no-repeat;
}
```

and then add your image to `/app/assets/images/background.jpg`